src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -186,14 +204,16 @@ def merge_evidence_qc_table(
     df_melt_insert_size = read_melt_insert_size(filename_melt_insert_size)
 
     # all data frames
-    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
-           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+    dfs = [df_ploidy, df_sex_assignments, df_bincov_median, df_wgd_scores, df_non_diploid,
+           df_manta_high_outlier, df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
            df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers,
            df_melt_insert_size]
     for df in dfs:
         df[ID_COL] = df[ID_COL].astype(object)
     output_df = reduce(lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs)
     output_df = output_df[output_df[ID_COL] != EMPTY_OUTLIERS]
+    output_df = output_df.replace([None, np.nan], 0.0)
```

Comment:
We do want to set samples who are not outliers in any categories to 0 in the outlier columns, rather than NaN, for interpretability. But I think it makes more sense to do that for just the outlier columns, not the entire dataframe. There may not be other metrics which routinely have NaNs at the moment, but if an NaN did show up in another metric (current or future), it would not necessarily make sense to set it to 0